### PR TITLE
feat: add input focus update emit

### PR DESCRIPTION
**Pull request by anasvakyathodi**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: src/color/Box.vue [hunk 1]
```diff
@@ -3,7 +3,12 @@
     <span class="name">
       {{ name }}
     </span>
-    <input v-model="modelColor" class="value" />
+    <input
+      v-model="modelColor"
+      class="value"
+      @focus="handleFocus(true)"
```
Comment (anish2690) [inline src/color/Box.vue:9]: @anasvakyathodi i think better to return the original html event here

Comment (anish2690) [reply src/color/Box.vue:9]: return focus, blur separately

Comment (anasvakyathodi) [reply src/color/Box.vue:9]: got it

Comment (anasvakyathodi) [reply src/color/Box.vue]: done ✅

**Pull request merged**